content/en/docs/getting-started/_index.md

```diff
@@ -60,73 +61,141 @@ olm-operator       1/1     1            1           5m52s
 packageserver      2/2     2            2           5m43s
 ```
 
-## Installing an Operator using OLM
 
-When you install OLM, it comes packaged with a number of Operators developed by the community that you can install instantly.
-You can use the `pacakagemanifest` api to see the operators available for you to install in your cluster:
+## Installing OLM and operator bundle via SDK for development purpose
```

Comment:
Another minor nit: I think an extra newline got added here somewhere when I look at the rendered tab for this. Can we remove that? It looks like line 63?